store/sqlstore/compliance_store.go

```diff
@@ -342,3 +342,55 @@ func (s SqlComplianceStore) MessageExport(cursor model.MessageExportCursor, limi
 	}
 	return cposts, cursor, nil
 }
+
+func (s SqlComplianceStore) BlocksExport(cursor model.BlockExportCursor, limit int) ([]*model.BlockExport, model.BlockExportCursor, error) {
+	var version int
+	schemaVersionRow := s.GetReplicaX().QueryRowX("select version from focalboard_schema_migrations")
+	if schemaVersionRow.Err() != nil {
+		return nil, cursor, schemaVersionRow.Err()
+	}
+	schemaVersionRow.Scan(&version)
+	if version != 16 {
+		return nil, cursor, fmt.Errorf("invalid focalboard version to export compliance, current=%d, expected=%d", version, 16)
```

Comment:
Nit: making it clearer that any version above 16 is fine.

```suggestion
		return nil, cursor, fmt.Errorf("invalid focalboard version to export compliance, current=%d, expected at least=%d", version, 16)
```

Reply:
If the user sees this error, how would they know which Focalboard plugin version to upgrade to? Should we use the Focalboard version in this error message? We can use the Focalboard plugin version we're expecting to ship this feature in.